Added Utility Fns and Uninstallation option.

semi710 opened the pull request:
added `check` fn to clean out the code.
and now install also handles uninstallation

semi710 committed:
Added Utility Fns and Uninstallation option.

added `check` fn to clean out the code.
and now install also handles uninstallation

K4R7IK commented on devexcuse.sh:
```diff
@@ -1,21 +1,37 @@
+	# Performs check if a command exists in the system
+	# sets the `$?` to 0 if the command exists, 1 otherwise
+	command -v "$1" &>/dev/null
+}
```
Could have integrated this in `check_dependencies` function

K4R7IK commented on devexcuse.sh:
```diff
@@ -1,21 +1,37 @@
 
-get_random_excuse(){
- jq -r '.[] | select(.text_en) | .text_en' $DATAFILE | shuf -n 1
+get_random_excuse() {
+	jq -r '.[] | select(.text_en) | .text_en' "$DATAFILE" | shuf -n 1
```
Random space added for no reason.

semi710 replied:
Done by shfmt (standard shell formatting)

K4R7IK reviewed (changes requested):
Integrate the check function in check_dependecies.

semi710 committed:
resolved reviews

K4R7IK committed:
Removed Checked function.

Didn't seem necessary to write a function for a single line of code. Hence removed it.
